Jenkinsfile

```diff
@@ -41,7 +41,9 @@ pipeline {
                         STORYBOOK = prDetails.labels.count { label -> label.name.equalsIgnoreCase('CI/storybook-url') }.equals(1)
                     }
 
-                    if (SKIP && PLATFORM != 'noplatform') {
+                    // TODO: linux-x64 should remain unskippable until we have a good way of automatically determining what really needs to run in a PR.
```

Comment:
Isn't it easier or move visible to just remove the skip label from GitHub?

Reply:
You're right. With this they'll only have the surprise that something runs with a skip label applied.